modules/backend/controllers/Auth.php

```diff
@@ -100,12 +100,14 @@ public function signin_onSubmit()
             'password' => post('password')
         ], $remember);
 
-        try {
-            // Load version updates
-            UpdateManager::instance()->update();
-        }
-        catch (Exception $ex) {
-            Flash::error($ex->getMessage());
+        if (config('cms.runMigrationsOnLogin', true)) {
```

Comment:
I would prefer that you used the Config facade here.

Reply:
@LukeTowers changed, and changed for another too :)